runtime/drivers/athena/warehouse.go

```diff
@@ -22,10 +22,15 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"github.com/rilldata/rill/runtime/drivers"
 	rillblob "github.com/rilldata/rill/runtime/drivers/blob"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 	"gocloud.dev/blob"
 	"gocloud.dev/blob/s3blob"
 )
 
+var tracer = otel.Tracer("github.com/rilldata/rill/runtime/drivers/unload")
```

Comment:
I think the tracer name is always supposed to match the package name
```suggestion
var tracer = otel.Tracer("github.com/rilldata/rill/runtime/drivers/athena")
```